safe_transaction_service/history/indexers/internal_tx_indexer.py

```diff
@@ -239,22 +233,22 @@ def _get_internal_txs_to_decode(
         self, tx_hashes: Sequence[str]
     ) -> Generator[InternalTxDecoded, None, None]:
         """
-        Use generator to be more RAM friendly
+        Retrieve relevant `InternalTxs` and if possible decode them to return `InternalTxsDecoded`
+
+        Every `InternalTx` must be retrieved again as `bulk_create` with `ignore_conflicts=True`
```

Comment:
I spend sometime trying to understand why you are mentioning here the bulk_create :smiling_face_with_tear: 
Maybe could be interesting to add that this functions is returning a generator to be used in a bulk_create_from_generator.

Reply:
Agree, I moved the comment after `InternalTx.objects.bulk_create_from_generator`